source/codegen/metadata/nidaqmx/custom_proto.mako

```diff
@@ -5,23 +5,23 @@ message AnalogPowerUpState {
 }
 
 message ChannelNamesAndExpState {
-  string channelNames = 1;
+  string lines = 1;
   DigitalLineState expState = 2;
 }
 
```

Comment:
These names are a little off (Especially PowerUpType) and inconsistent in where channel/state/function are in the name. Since it's kind of obnoxious to change them, I want to get on the same page before fixing them.

I like to keep "channel" before "state". I think it makes sense to use "Channels" when they accept multiple (and that should match "channelName" vs "channelNames"). I think it's more discoverable with function at the beginning although it's more affinitized to the state than the channel:

AnalogPowerUpChannelsAndState
WatchdogExpChannelsAndState
DigitalPowerUpChannelAndState
DigitalPowerUpChannelsAndState 
DigitalPullIUpPullDownChannelsAndState

Does that seem OK?

Reply:
Yeah, I like that. Thanks for consistent-ifying these!